asp_interactive_configuration/ooasp/interactive.py

```diff
@@ -12,8 +12,11 @@
 from copy import deepcopy
 import ooasp.utils as utils
 from typing import List
-import ooasp.encodings
-import ooasp.encodings_simple
+import ooasp.settings as settings
+
+def log(x):
+    # print(x)
+    return
```

Comment:
Please add a docstring, e.g., "Used for logging if print(x) is not commented out"